addons/website_sale/models/product_template.py

```diff
@@ -548,7 +544,11 @@ def _get_additionnal_combination_info(self, product_or_template, quantity, date,
             'taxes': taxes,  # taxes after fpos mapping
         })
 
-        if pricelist.discount_policy != 'without_discount':
+        if pricelist._get_product_rule_policy(
+            product=product_or_template,
+            quantity=quantity,
+            target_currency=currency,
+        ) != 'percentage':
             # Leftover from before cleanup, different behavior between ecommerce & backend configurator
             # probably to keep product sales price hidden from customers ?
             combination_info['list_price'] = combination_info['price']
```

Comment:
This part could be dropped if discount computation is harmonized between backend and fronted (cf other comment).